php/commands/core.php

```diff
@@ -1150,6 +1164,29 @@ private function get_updates( $assoc_args ) {
 		return array_values( $updates );
 	}
 
+	private function _maybe_remove_unused_files( $version_from, $version_to) {
+		if ( ! $version_from || ! $version_to ) {
+			return;
+		}
+
+		$files_to_remove = array();
+		$includes_folder = defined( 'WPINC' ) ? WPINC : '/wp-includes';
+
+		if ( version_compare( $version_from, '4.4', '>=' ) && version_compare( $version_to, '4.4', '<' ) ) {
+			$files_to_remove[] = ABSPATH . $includes_folder . '/embed-functions.php';
+			$files_to_remove[] = ABSPATH . $includes_folder . '/class-wp-oembed-controller.php';
+			$files_to_remove[] = ABSPATH . $includes_folder . '/rest-api.php';
+		}
+
+		if ( ! empty( $files_to_remove ) ) {
+			foreach ( $files_to_remove as $file ) {
+				if ( file_exists( $file ) ) {
+					unlink( $file );
```

Comment:
This could potentially be a lot of files. Would it better to just output a count of files?